### DISCUSSION TITLE
[Kavita] Smart filters opened from dashboard should respect dashboard restrictions.

### DISCUSSION THREAD

**Discussion by kayhos** [Kareadita/Kavita#3149] [Ideas]
### Idea Description

I would like to have the same behavior from the smart filters that are opened from the dashboard (recently updated Series and newly added series).

When launching those filters, a user expects the behavior from the dashboard to continue as you end up with a smart filter that shows a totally different list of series from the list displayed in the dashboard.

My current use case is where some libraries have material that should not appear on the dashboard due to being too graphic or unsuitable to be displayed openly. The user still needs to have access to those libraries but does not want to display their content.

### Idea Category

Web UI

### Duration of Using Kavita

6 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by scare376** [comment]
This sounds like what you want is similar to #2536

**Reply by scare376** [reply]
#2766 could be relevant as well

**Reply by kayhos** [reply]
They are indeed similar, but my feature request / idea is to align the expected behavior when launching a view from the Dashboard, the smart filter should show the same time as the content displayed on the dashboard. 

For example, I have 4 libraries that are not displayed on the dashboard but they have fairly frequent content updates, multiple times in a day. These content updates push back the one that I really want to see on the Dashboard page that are I feel are more important.

**Comment by kayhos** [comment]
After having a chat with Horo on discord ( https://discord.com/channels/821879810934439936/821879810934439939/1285062556062580746 ), I believe that adding a filtering value like "Dashboard visible" and including that filtering in the default filters (On Hold, Recently Updated Series, Newly Added Series and More In) might: 

- Allow for a more coherent behavior
- Be simpler to implement
- Allow this kind of control for custom Smart Filters too.

**Reply by majora2007** [reply]
You will need to take this new comment and move to a new discussion. I have already implemented the original ask and will be closing the discussion with the release.

Btw, the dashboard streams (and now anything used from dashboard, have the correct context applied to respect library media inclusion settings.

**Comment by kayhos** [comment]
Well in this case, this covers my need, do you still want me to add a feature request for the "Displayed on Dashboard" filtering value ? I don't feel like I need that anymore with your implementation.

**Reply by majora2007** [reply]
That's for you to decide. I would wait till after the stable for you to assess then if you feel more enhancement is needed, you can open a new FR.
